custom_components/plugwise/climate.py

```diff
@@ -137,7 +144,7 @@ def hvac_mode(self) -> HVACMode:
         return HVACMode(mode)
 
     @property
-    def hvac_action(self) -> HVACAction:
+    def hvac_action(self) -> HVACAction | None:
```

Comment:
None is not needed, always a HVACAction is returned.
Change in Core instead.

Reply:
Re-tagged